.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,102 @@
+---
+name: Bug report
+description: Create a BUG report.
+title: "[BUG]: Title"
+labels: Bug
+body:
+- type: markdown
+  attributes:
+    value: |
+      Before you submit this request, please be aware that this is an
+      open source project, so you are invited to become a contributor to this project.
+
+      During reproduction please enable all logs e.g. ledctl --log-level=all --list-controllers
```

Comment:
Imo "For reproduction" is better suited, meaning it should be done before reproduction, not in the middle. But maybe it's just me being picky, I leave this up to you.